### PR TITLE
Added method urls to all registers

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: data_ocean/fixtures/register.json [hunk 1]
```diff
@@ -59,6 +67,8 @@
         "name": "Адресний реєстр",
         "source_name": "drv.gov.ua",
         "source_register_id": "",
+        "data_ocean_list": "/api/drvbuilding/",
+        "data_ocean_retrieve": "/api/drvbuilding/{drvbuilding_id}",
```
Comment (roman-tiukh) [inline data_ocean/fixtures/register.json:71]:
```suggestion
        "data_ocean_retrieve": "/api/drvbuilding/{id}/",
```

File: data_ocean/fixtures/register.json [hunk 2]
```diff
@@ -20,6 +22,8 @@
         "name": "Класифікація видів економічної діяльності",
         "source_name": "data.gov.ua",
         "source_register_id": "e1afb81c-70e4-4009-96a0-b240c36e4603",
+        "data_ocean_list": "/api/kved/",
+        "data_ocean_retrieve": "/api/kved/{kved_id}",
```
Comment (roman-tiukh) [inline data_ocean/fixtures/register.json:26]:
```suggestion
        "data_ocean_retrieve": "/api/kved/{id}/",
```

File: data_ocean/fixtures/register.json [hunk 3]
```diff
@@ -33,6 +37,8 @@
         "name": "Реєстр адміністративно-територіального устрою",
         "source_name": "data.gov.ua",
         "source_register_id": "a2d6c060-e7e6-4471-ac67-42cfa1742a19",
+        "data_ocean_list": "/api/street/",
+        "data_ocean_retrieve": "/api/street/{street_id}",
```
Comment (roman-tiukh) [inline data_ocean/fixtures/register.json:41]:
```suggestion
        "data_ocean_retrieve": "/api/street/{id}/",
```

File: data_ocean/fixtures/register.json [hunk 4]
```diff
@@ -7,6 +7,8 @@
         "name": "Єдиний державний реєстр юридичних осіб, фізичних осіб-підприємців та громадських формувань",
         "source_name": "data.gov.ua",
         "source_register_id": "1c7f3815-3259-45e0-bdf1-64dca07ddc10",
+        "data_ocean_list": "/api/company/",
+        "data_ocean_retrieve": "/api/company/{company_id}",
```
Comment (roman-tiukh) [inline data_ocean/fixtures/register.json:11]:
```suggestion
        "data_ocean_retrieve": "/api/company/{id}/",
```

File: data_ocean/models.py [hunk 5]
```diff
@@ -29,6 +29,8 @@ class Register(DataOceanModel):
     name = models.CharField(max_length=500, unique=True)
     source_name = models.CharField(max_length=300)
     source_register_id = models.CharField(max_length=36, unique=True)
+    data_ocean_list = models.URLField(max_length=500)
+    data_ocean_retrieve = models.URLField(max_length=500)
```
Comment (roman-tiukh) [inline data_ocean/models.py:33]:
```suggestion
    data_ocean_list = models.URLField(max_length=500, null=True, blank=True, default=None)
    data_ocean_retrieve = models.URLField(max_length=500, null=True, blank=True, default=None)
```

File: data_ocean/fixtures/register.json (hunk 2 above)
Comment (roman-tiukh) [inline data_ocean/fixtures/register.json:26]: you missed one

File: data_ocean/fixtures/register.json [hunk 6]
```diff
@@ -46,6 +52,8 @@
         "name": "Класифікатор об’єктів адміністративно-територіального устрою України",
         "source_name": "data.gov.ua",
         "source_register_id": "dc081fb0-f504-4696-916c-a5b24312ab6e",
+        "data_ocean_list": "/api/koatuu",
+        "data_ocean_retrieve": "/api/koatuu/{id}",
```
Comment (IegorSoboliev) [inline data_ocean/fixtures/register.json:56]: get new instruction)

**Pull request merged**